DietScoopApp/app/src/main/java/com/example/dietscoop/Database/IngredientStorage.java

```diff
@@ -1,21 +1,14 @@
 package com.example.dietscoop.Database;
 
-import android.os.Build;
 import android.util.Log;
 
-import androidx.annotation.Nullable;
-import androidx.annotation.RequiresApi;
-
+import com.example.dietscoop.Activities.IngredientListActivity;
 import com.example.dietscoop.Data.Ingredient.IngredientCategory;
 import com.example.dietscoop.Data.Comparators.IngredientComparator;
 import com.example.dietscoop.Data.Ingredient.IngredientInStorage;
 import com.example.dietscoop.Adapters.IngredientStorageAdapter;
-import com.example.dietscoop.Fragments.sortIngredientByFragment;
```

Comment:
Please include these imports for the next pr.

Reply:
they are unused well be fine without them, I think android studio deleted them automatically